h2/src/main/org/h2/tools/Csv.java

```diff
@@ -834,6 +865,8 @@ public String setOptions(String options) {
                 setLineSeparator(value);
             } else if (isParam(key, "null", "nullString")) {
                 setNullString(value);
+            } else if (isParam(key, "quotedNulls", "quotedNullsBoolean")) {
```

Comment:
`quotedNullsBoolean` isn't useful here, `quotedNulls` is enough.

Reply:
Removed.